dotCMS/src/main/java/com/dotmarketing/business/PermissionBitAPIImpl.java

```diff
@@ -1406,6 +1406,7 @@ private boolean isFolder(final Permissionable permissionable) {
 	private boolean isHost(final Permissionable permissionable) {
 
 		return permissionable instanceof Host ||
+				(permissionable instanceof Contentlet && Host.class.getSimpleName().equals(((Contentlet) permissionable).getContentType().name())) ||

```

Comment:
I think we should use the `contentlet.isHost()` method here rather than recreating it

Reply:
done https://github.com/dotCMS/core/pull/17906/commits/0eabafde4729f91ac99fece796c7d50cbaba5f40